nixos/modules/services/networking/bitlbee.nix

```diff
@@ -171,6 +171,7 @@ in
         description = "BitlBee user";
         home = "/var/lib/bitlbee";
         createHome = true;
```

Comment:
If this service doesn't a lot of files, switching over to systemd's `DynamicUser` would be preferred.
If you enable that systemd then takes care of creating the state directory and picking up a uid/gid.

Reply:
@lheckemann && @rnhmjoj I think this is a perfect fit for this.
Unsure if this will break existing setups, but I will try `DynamicUser` when I get the time to do so.

Anyone else using Bitlbee feel free to do so as well!

Reply:
> Unsure if this will break existing setups, but I will try DynamicUser when I get the time to do so.

It shouldn't AFAIK. systemd always tries to reuse the last uid/gid if available, so existing setup should continue to use the same (fixed) id as before. If for some reason it isn't, it automatically updates the file ownership to match the new (generated) one. (This is way `DynamicUser` shouldn't be used if you have a lot of state files)